database.js

```diff
@@ -0,0 +1,10 @@
+import admin from "firebase-admin";
+const serviceAccount = JSON.parse(process.env.FIREBASE_SERVICE_ACCOUNT);
+
+admin.initializeApp({
+  credential: admin.credential.cert(serviceAccount),
+  databaseURL: "https://yu-gi-oh-inventory.firebaseio.com",
+});
+
+const db = admin.database();
+export default db;
```

Comment:
by convention, even though it doesn't actually affect anything, the default export is usually named the same as the file name. So in this case, `export default database`

Reply:
alternatively you can rename this file to `db.js`, either are totally fine